menu-bar/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+      <meta charset="utf-8" />
+      <title>Sway</title>
+      <link rel="stylesheet" href="styles.css" />
+      <link rel="stylesheet" href="fonts.css" />
+  </head>
+  <body>
+      <header>
+          <a href=""><img class="sway-logo-resize" src="swaylogo2.png"
```

Comment:
Double-check all of your indentation. Some of it is 4 spaces and not all tag pairs are matched.